### ISSUE TITLE
Clearly mark project as unmaintained

### ISSUE THREAD

**Issue by wilhelmy** [jsongraph/jgf-dot#9]
After some digging, I've discovered that [graphlib-dot](https://github.com/dagrejs/graphlib-dot) is unmaintained, and jgf-dot only supports JSON Graph Format version 1.

Maybe it would make sense to let everyone know about these facts?

**Comment by wshayes** [comment]
Good idea and done. Unfortunately, I'm a lot better at Python than Javascript, and I haven't heard from @RangerMauve who originally created and maintained this project.

**Comment by RangerMauve** [comment]
Hey folks! I've been focused on other stuff but json graphs still have a spot in my heart. 😁

I don't know about what we should do for replacing graphlib-dot (if that's even necessary), but I'd be down to update this repo if there's specific tasks that need to be done. 👍

**Comment by wilhelmy** [comment]
Hi! As far as I'm concerned graphlib-dot being unmaintained/deprecated (in favour of what exactly else they don't say, so maybe this is once more the kind of javascript ecosystem problem we've heard so much about ;) doesn't bother me (after all I'm not a maintainer here) but missing JGFv2 support does. Now I could change my code to additionally support JGFv1, but I actually already have a graphviz output that I wrote before learning about JGF's general existence. So I stumbled over this, and it seems so did the rest of people who opened issues here in the past.

Not to put pressure on you, we're all mostly still maintaining FOSS stuff in our spare time. Glad to hear you still have a spot in your heart for old projects ;)
